tests/Application/src/Grid/Builder/AttributeNationalityFilter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Grid\Builder;
+
+use App\Entity\Nationality;
+use App\Grid\Type\NationalityFilterType;
+use Sylius\Bundle\GridBundle\Builder\Filter\Filter;
+use Sylius\Bundle\GridBundle\Builder\Filter\FilterInterface;
+
+final class AttributeNationalityFilter
+{
+    public static function create(string $name, ?bool $multiple = null, ?array $fields = null): FilterInterface
+    {
+        $filter = Filter::create($name, $name);
```

Comment:
I think we can directly call NationalityFilter::create method.

Reply:
The NationalityFilter is the example that's here to ensure using ConfigureableFilter still works, they are not linked together.
NationalityFilter actually use Filter::create() method.

Reply:
Improved with ::class const